amr-wind/incflo.H

```diff
@@ -58,6 +62,11 @@ public:
     // Evolve solution to final time through repeated calls to Advance()
     void Evolve();
 
+#ifdef ERF_AW_MULTIBLOCK
+    // Evolve solution in a multiblock framework by desired number of steps
+    void Evolve_MB(int MBstep, int max_block_step);
```

Comment:
const these and change `MBstep` to `multiblock_step`. Same for the `Evolve_MB` to `Evolve_MultiBlock`. It's nice to have explicit names.